ShopApp/ShopApp/Controller/PersonalDataViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  PersonalDataViewController.swift
+//  ShopApp
+//
+//  Created by Andreea Sburlea on 23.06.2022.
+//
+
+import UIKit
+import FirebaseAuth
+import Firebase
+
+class PersonalDataViewController: UIViewController {
+
+    @IBOutlet private var nameLabel: UILabel!
+    @IBOutlet private var emailLabel: UILabel!
+    @IBOutlet private var photoImageView: UIImageView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        photoProperties()
+        getUserInfo()
+    }
+
+    func photoProperties() {
```

Comment:
All method names should contain a verb. This one can be like "setPhotoProperties"